src/components/Composer.vue

```diff
@@ -463,7 +463,7 @@ export default {
 
 		// Add attachments in case of forward
```

Comment:
please add a note about why we exclude inline attachments here

Reply:
done